spec/features/school_student/creating_a_batch_of_school_students_spec.rb

```diff
@@ -46,17 +63,17 @@
     expect(ProfileApiClient).not_to have_received(:create_safeguarding_flag).with(token: UserProfileMock::TOKEN, flag: ProfileApiClient::SAFEGUARDING_FLAGS[:teacher], email: owner.email)
   end
 
-  it 'responds 204 No Content' do
+  it 'responds 202 No Content' do
```

Comment:
```suggestion
  it 'responds 202 Accepted' do
```

Reply:
Good spot on this and the other, removed! thanks